CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
```

Comment:
Will 3.6.0 be released or will we replace this changelog / release version 4.0.0 directly.